javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
On one hand we'd expect to use it as `JSON.stringify`, but indeed it's easier to check for `undefined` then to wrap it in `try-catch`. I'll add the condition in the inspector.